src/components/header/MegaMenu/MegaMenu.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import { List } from '../List/List'
+
+type MegaMenuProps = {
+  items: React.ReactNode[][]
+  isOpen: boolean
+}
+
+export const MegaMenu = (props: MegaMenuProps): React.ReactElement => {
+  const { items, isOpen } = props
+  return (
+    <div className="usa-nav__submenu usa-megamenu" hidden={!isOpen}>
+      <div className="grid-row grid-gap-4">
```

Comment:
hmmmmmmm, I'm not quite sure I agree. I think the point of the menu is to be able to take in a grid ([][]) of items to display in a list. I don't know of the top of my head what other use it would have.

Reply:
🤔  I think we have different understanding on how flexible or not these subcomponents should be.  Originally I thought the point of these sub-components was to encapsulate  already opinionated styles and markup for optional reuse inside a USWDS standard parent component (`Header` or  `Footer`).  I thought they would intentionally be prescriptive about layout/style - and closely imitate the examples in their docs.

Maybe we need to create a light guideline for when we export a reusable sub-component in this library  ...  and when we just make an "example" component for use a story

Reply:
Summary of our convo today:

- We want to strike a balance between providing "drop-in" components that can implement USWDS styles using their guidelines, and also keeping them from being too prescriptive for different use cases.
- For now, when in doubt defer to the USWDS docs and treat them as requirements. We can start by covering their basic examples as test cases, and iterate on them to make them more flexible if needed
- Going forward, we should make sure to include acceptance criteria on issues so we can discuss components & agree on requirements prior to building

Reply:
Given the above comments, for the first round, I am going to leave the component as rendering a list of items as described in the USWDS guidelines.